### DISCUSSION TITLE
EIA923 - out_eia__monthly_generators vs. core_eia923__monthly_generation

### DISCUSSION THREAD

**Discussion by Ylawy** [catalyst-cooperative/pudl#3387] [Help Me!] [answered]
Hi there,

I have been looking at your data dictionaries to get a better look at the data from the EIA923 form and there is a lot of data available which is wonderful. So thank you for making this available to the public. However, I noticed that there are 2 data types related to monthly generation data, which are 
core_eia923__monthly_generation and out_eia__monthly_generators, and I'm confused as to what the difference in those actually is. data series out_eia__monthly_generators seems more granular, but they seem to hold similar data or am I wrong? What exactly do the "out" and "core" stand for?

Also, in terms of heat rate, what would be the best data series to form an opinion for a particular region on monthly granularity (let's say Pennsylvania)? At the moment there are 22 different data series containing the term "heat_rate" that I'm a bit overwhelmed what data series is the right one.

Thank you very much.

**Comment by bendnorman** [comment, marked as answer]
Great questions! Generally, you should refer to tables with the `out` prefix. The `core` tables are joined together to create the denormalized `out` tables which are easier to work with. You can read more about the table naming conventions [in the documentation](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/naming_conventions.html).

The `out_eia__monthly_generators` table is our attempt at collating most generator attributes into one table. It contains heat rate and net generation information, see columns: `net_generation_mwh` and `unit_heat_rate_mmbtu_per_mwh`.

`core_eia923__monthly_generation` is one table that feeds into the final `out_eia__monthly_generators` table. All generators in `core_eia923__monthly_generation` exist in `out_eia__monthly_generators`.
